src/client/Client.cc

```diff
@@ -14743,6 +14754,61 @@ mds_rank_t Client::_get_random_up_mds() const
   return *p;
 }
 
+mds_rank_t Client::hash_into_mds_rank(inodeno_t ino)
+{
+  std::set<mds_rank_t> up;
+  mdsmap->get_up_mds_set(up);
+  if (up.empty())
+    return MDS_RANK_NONE;
+  const mds_rank_t max_mds = up.size();
```

Comment:
max_mds = mdsmap->get_max_mds() 

https://github.com/ceph/ceph/blob/c06e46239e781d519e1f80a43e208fb521a8b554/src/mds/MDCache.cc#L900

I perfer to define a common function.